File: CSharp-Part-1/3. Operators-and-Expressions/homework/08. Prime Check/PrimeCheck.cs
using System;

namespace _08.Prime_Check
{
    class PrimeCheck
    {
        static void Main()
        {
            uint intToCheck;
            int count = 0;
            Console.WriteLine("Provide a positive integer <= 100 to check:");
            while (!(uint.TryParse(Console.ReadLine(), out intToCheck) && (intToCheck <= 100)))
            {
                Console.WriteLine("false");
            }
            if (intToCheck != 1)
            {
                for (int i = 2; i <= Math.Sqrt(intToCheck); i++)
                {
                    count = count + (((intToCheck % i) == 0) ? 1 : 0);
                }
            }
            else
            {
                count = 2;
            }

            Console.WriteLine((count > 0) ? "Is Not Prime" : "Is Prime");
            Console.WriteLine("\nPress any key to exit");
            Console.ReadKey();
        }
        }
    }

File: 2. Web Design and UI technologies/3. JavaScript-Fundamentals/02. Operators and Expressions/07. Prime Check/PrimeCheck.js
function PrimeCheck (args) {
  var n = +args[0];
  
  if (n <= 1) {
  console.log("false");
  }
  else {
    var primeCount = 0;
      for (var i = 1; i <= n; i++) {
        if (n % i === 0) {
          primeCount++;
        }
      }
      if (primeCount > 2) {
        console.log("false");
      }
      else {
        console.log("true");
      }
  }
}